backend/handlers/handlers.go

```diff
@@ -515,17 +518,21 @@ func (handlers *Handlers) getAccountsTotalBalanceHandler(_ *http.Request) (inter
 		} else {
 			totalPerCoin[coinCode] = new(big.Int).Add(totalPerCoin[coinCode], amount.BigInt())
 		}
+		conversionsPerCoin[coinCode] = coin.Conversions(coin.NewAmount(totalPerCoin[coinCode]), account.Coin(), false, account.Config().RateUpdater)
 	}
 
-	formattedTotalPerCoint := make(map[coin.Code]string)
 	for k, v := range totalPerCoin {
 		currentCoin, err := handlers.backend.Coin(k)
 		if err != nil {
 			return nil, err
 		}
-		formattedTotalPerCoint[k] = currentCoin.FormatAmount(coin.NewAmount(v), false)
+
+		totalAmount[k] = accountHandlers.FormattedAmount{
+			Amount:      currentCoin.FormatAmount(coin.NewAmount(v), false),
+			Conversions: conversionsPerCoin[k],
```

Comment:
maybe i am blind, but is this used anywhere? the `ITotalBalance` interface in the frontend is unchanged, and it doesn't fit `IAmount`.

It seems everything works also without adding `Conversions` here, not sure how we came to the conclusion that this was needed here :sweat_smile: do you know?

Reply:
I think this is correct. Why do you say that it doesn't fit IAmount? It is used here, to display the fiat value of the total coin balance for multiple accounts https://github.com/digitalbitbox/bitbox-wallet-app/pull/1735/commits/f785a580e1261cf545337236b55aa1ab15028db9#diff-e5d06f77122fc6a06ea5e7c80d3466cd0fa4d69a252e8108de78062a5bdbea83R255

If you don't add `Conversions` in `getAccountsTotalBalanceHandler` return value, it won't be correctly displayed.